swagger_client/models/conditions.py

```diff
@@ -191,7 +191,7 @@ def probe_responses(self, probe_responses):
     def character_vitals(self):
         """Gets the character_vitals of this Conditions.  # noqa: E501
 
-        True if the specified list of vitals values have been met for the specified character_id  # noqa: E501
+        True if the any of the specified collection of vital values have been met for the specified character_id  # noqa: E501
```

Comment:
Pretty sure this is just carried over when the model was generated, but "the any of the", again.